src/cockpit/beiboot.py

```diff
@@ -131,10 +131,25 @@ class AuthorizeResponder(ferny.AskpassHandler):
     commands = ('ferny.askpass', 'cockpit.report-exists')
     router: Router
 
-    def __init__(self, router: Router):
+    def __init__(self, router: Router, basic_password: Optional[str]):
         self.router = router
+        self.basic_password = basic_password
+        self.have_basic_password = bool(basic_password)
```

Comment:
This would I presume break auth when a user has an empty password, I am not sure if that is supposed to be supported. That is my point, because:

`bool("") => False`

Reply:
Ah, I see -- I suppose I could add a test case for an empty password (not sure if that works with ssh), and then do the above condition as "user **and** password are empty → user_password = None". I don't know if it's supported as we never tested that anywhere.

Reply:
It works with ssh with

    echo "PermitEmptyPasswords yes" > /etc/ssh/sshd_config.d/01-empty-password.conf

And then I can also log into Cockpit with beiboot. 

This works because there isn't any actual prompt from ssh for a password, it logs you straight in.

So `""` never actually appears in this code path, and thus `is not None` just works as well. Changed accordingly.